scripts/create-patch.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Does this need to be a script?  Could it be just a yaml manifest in the custom-cert-webhook directory?

Reply:
No. I thought  it was not a good idea to change the dir structure. I will add a manifest in custom-cert-webhook